docker-compose.yml

```diff
@@ -27,7 +27,9 @@ services:
       - env.d/development/${DEV_ENV_FILE:-dev}
       - env.d/development/${DB_HOST:-postgresql}
     networks:
-      - default
+      default:
```

Comment:
Actually, you can declare different names depending on the network like we do here: https://github.com/openfun/openedx-docker/blob/master/docker-compose.yml#L180

This should allow you to keep app for the default name which is better?

Reply:
The problem is that Joanie django application service name is also `app` and this service is exposed on the `edx_outside` network. Of course, I expose Joanie on the `edx_outside` with an alias `joanie`, but it seems the service is accessible from `app` and `joanie` host names on this network.

From Richie, the `nginx` needs to resolve the `app` host name and the conflict appears here:
- On `default` network, it can resolve the Richie `app` service
- On `edx_outside` network, it can resolve the Joanie `app` service

So I have two solutions:
- Rename the joanie `app` service
- Use an alias to the richie `app` service on `default` network (Solution I implemented here)